docs/UserGuide.md

```diff
@@ -2,53 +2,43 @@
 layout: page
 title: User Guide
 ---
-# TutorBuddy
-Welcome to the user guide for TutorBuddy!<br/>
-Are you a tuition teacher looking for a reliable way to keep track of
-your 1-to-1 lessons? Needing a system to remind you about upcoming lessons?
```

Comment:
MIght be good to mention somewhere its 1-to-1 only

Reply:
Good point, ill add it in